src/utils/utils_windows_common.c

```diff
@@ -33,3 +34,13 @@ size_t util_get_page_size(void) {
 int utils_getpid(void) { return GetCurrentProcessId(); }
 
 int utils_gettid(void) { return GetCurrentThreadId(); }
+
+int utils_close_fd(int fd) { return -1; }
+
+umf_result_t utils_duplicate_fd(int pid, int fd_in, int *fd_out) {
+    // TODO: find another way to obtain a duplicate of another process's file descriptor
```

Comment:
A strange inadequate comment.

Reply:
removed.

P.S.: this comment was in the original function `os_duplicate_fd` that I just renamed and moved to utils.